features/admin.users.v1/pages/users.tsx

```diff
@@ -221,6 +225,32 @@ const UsersPage: FunctionComponent<UsersPageInterface> = (
         !isSubOrganization()
     );
 
+    // Combine the search query and account status filters query.
+    const combinedUserFilter: string = useMemo(() => {
+        let baseFilter: string | null = searchQuery === "" ? null : searchQuery;
+
+        if (selectedAccountStatusFilters.length > 0) {
+            const accountStatusScimFilters: string[] = selectedAccountStatusFilters.map((filterKey: string) => {
```

Comment:
rename to `scimFilterExpressions `

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/8171/commits/a35f3f308db99f0f624552fce3446dbdefa8b471